cfme/tests/middleware/test_middleware_deployment.py

```diff
@@ -1,33 +1,48 @@
-from random import sample
-
 import pytest
+from cfme.middleware import get_random_list
 from cfme.middleware.deployment import MiddlewareDeployment
+from cfme.middleware.server import MiddlewareServer
 from utils import testgen
 from utils.version import current_version
 
 pytestmark = [
-    pytest.mark.usefixtures('single_middleware_provider'),
+    pytest.mark.usefixtures('setup_provider'),
```

Comment:
single provider will not resolve that case, for this we need other solution, probably will not even have,
but my intension is in other: 
1) comparing UI with DB, without knowing whether all these values are correct or not, do not relay on other tests, maybe they are disabled or failing for some other reason
2) make sure that we know what is happening in MIQ, we own single provider which we have just created, and have all tests based on assured info (this is general rule)